src/safeds/data/tabular/containers/_table.py

```diff
@@ -2380,3 +2382,41 @@ def __dataframe__(self, nan_as_null: bool = False, allow_copy: bool = True):  #
         data_copy = self._data.reset_index(drop=True)
         data_copy.columns = self.column_names
         return data_copy.__dataframe__(nan_as_null, allow_copy)
+
+    def into_dataloader(self, batch_size: int) -> DataLoader:
+        """
+        Return a Dataloader for the data stored in this table, used for training neural networks.
+
+        The original table is not modified.
+
+        Parameters
+        ----------
+        batch_size : int
+            The size of data batches that should be loaded at one time.
+
+        Returns
+        -------
+        result : DataLoader
```

Comment:
```suggestion
        result :
```

The colon is needed for results.